class_generator/tests/manifests/api_server/api_server_res.py

```diff
@@ -18,7 +18,7 @@ class APIServer(Resource):
 
     def __init__(
         self,
-        additional_cors_allowed_origins: Optional[Dict[str, Any]] = None,
```

Comment:
how this changed from Dict to List?

Reply:
it is a list (defined as `<[]string>`, before we defaulted to dict if we did not have it. I added this as another type